apps/sms/views/shared/js/utils.js

```diff
@@ -522,10 +522,34 @@
     params: function(input) {
       var parsed = {};
       input.replace(rparams, function($0, $1, $2) {
+        if ($2 === 'true') {
```

Comment:
__question:__ mmm if add exception for booleans, can't we do the same for numbers with ```isFinite```? Btw, is booleans case for ```focusComposer``` mainly?